File: docs/Acceleration/color/rgb/yellow_color_generator.js
class YellowColorGenerator extends RGBColorGenerator {

    getRandomColor() {
        colorMode(RGB, 255);
        let r = int(random(100, 255));
        let g = int(random(100, 255));
        let b = int(random(50, 100));
        return color(r, g, b);
    }
}

File: Collisions/src/color/RGB/YellowColorGenerator.java
package color.RGB;

import processing.core.PApplet;

public class YellowColorGenerator extends RGBColorGenerator {

    public YellowColorGenerator(PApplet p) {
        super(p);
    }

    public int getRandomColor() {
        p.colorMode(PApplet.RGB, 255);
        int r = (int)p.random(100, 255);
        int g = (int)p.random(100, 255);
        int b = (int)p.random(50, 100);
        return p.color(r, g, b);
    }

}
